tests/test_models.rs

```diff
@@ -854,6 +854,267 @@ fn test_current_playback_context() {
     assert!(current_playback_context.progress.is_none());
 }
 
+#[test]
```

Comment:
Thanks for your contribution, is it possible to minimize the Json data, for example, reduce multiple objects in an  array to a single object?